Xamarin.PropertyEditing/ViewModels/PropertiesViewModel.cs

```diff
@@ -462,7 +462,7 @@ private PropertyViewModel GetViewModelCore (IPropertyInfo property)
 
 		private Task busyTask;
 
-		private static readonly Dictionary<Type,Func<TargetPlatform,IPropertyInfo,IEnumerable<IObjectEditor>,PropertyViewModel>> ViewModelMap = new Dictionary<Type, Func<TargetPlatform, IPropertyInfo, IEnumerable<IObjectEditor>, PropertyViewModel>> {
+		internal static readonly Dictionary<Type,Func<TargetPlatform,IPropertyInfo,IEnumerable<IObjectEditor>,PropertyViewModel>> ViewModelMap = new Dictionary<Type, Func<TargetPlatform, IPropertyInfo, IEnumerable<IObjectEditor>, PropertyViewModel>> {
```

Comment:
We don't need to add the mac specific types back in so this can stay private.

Reply:
Ok